src/middlewared/middlewared/alembic/versions/22.12/2022-08-09_13-37_privilege.py

```diff
@@ -0,0 +1,64 @@
+"""Account privileges
+
+Revision ID: 79942334975f
+Revises: 9b7127606e2b
+Create Date: 2022-08-09 13:37:57.243188+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '79942334975f'
+down_revision = '9b7127606e2b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('account_privilege',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('builtin_name', sa.String(length=200), nullable=True),
+    sa.Column('name', sa.String(length=200), nullable=False),
+    sa.Column('local_groups', sa.TEXT(), nullable=False),
+    sa.Column('ds_groups', sa.TEXT(), nullable=False),
```

Comment:
Does it make sense to have two different lists here since from OS standpoint they're just GIDs?

Reply:
Yes, I want a local fallback in cache DS is not reachable, that's why local groups must be explicitly specified for users (in most cases, one user) who is a local administrator.